src/llm-adapters.ts

```diff
@@ -252,6 +253,89 @@ function getClaudeAdapter(
   return adapter;
 }
 
+function getClaudeAdapter(anthropic: Anthropic, params?: CommonLLMParameters) {
+  const DEFAULT_CLAUDE_PARAMS = {
+    model: 'claude-2',
+    temperature: 0,
+  };
+
+  const DEFAULT_CLAUDE_LLM_CONFIG: LLMConfig = {
+    enableTodaysDate: true,
+    fewShotLearning: [],
+  };
+
+  const adapter: {
+    llmConfig: LLMConfig;
+    callLLM: LLMCallFunc;
+  } = {
+    llmConfig: DEFAULT_CLAUDE_LLM_CONFIG,
+    callLLM: async function callLLM(
+      messages: LLMCompatibleMessage[],
+      queryPrefix?: string,
+    ) {
+      try {
+        const formattedMessages = messages.filter(
+          (message) => message.role !== 'system',
```

Comment:
you need to remove any trailing spaces with a `trimEnd` from the last message if it's from the assistant - something weird Claude has now

Reply:
I'll trim it from all assistant messages, irrespective of whether it's the last. that shouldn't be a problem right?

Reply:
Yup that works too